tests/integration-tests/conftest.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+import os
+import re
+from shutil import copyfile
+
+import pytest
+
+from clusters_factory import ClustersFactory
+from conftest_markers import (
+    DIMENSIONS_MARKER_ARGS,
+    add_default_markers,
+    check_marker_dimensions,
+    check_marker_list,
+    check_marker_skip_dimensions,
+    check_marker_skip_list,
+)
+from jinja2 import Environment, FileSystemLoader
+
+
```

Comment:
I'd add a comment on top to understand the purpose of this important file.

Reply:
done!